src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -95,6 +92,36 @@ public static Email parseEmail(String email) throws ParseException {
         return new Email(trimmedEmail);
     }
 
+    /**
+     * Parses a {@code String ic} into an {@code Ic}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code ic} is invalid.
+     */
+    public static Ic parseIc(String ic) throws ParseException {
+        requireNonNull(ic);
+        String trimmedIc = ic.trim();
```

Comment:
add .toUpperCase() also in case the user enters a lowecase ic number for consistency